auslib/test/web/test_client.py

```diff
@@ -656,7 +681,13 @@ def testVersion2Get(self):
 
     def testVersion2GetIgnoresRuleWithDistribution(self):
```

Comment:
Should we rename the function, now that it doesn't return an empty response anymore?

Reply:
Good idea